packages/tests/tests/filter.node.spec.ts

```diff
@@ -2,7 +2,9 @@ import {
   createDecoder,
```

Comment:
Amazing job! I have only one comment here - the file is huge now. 
I propose to break it into some files under `filter` folder.

Reply:
Yes, makes sense, I'll create a folder with multiple files based on features:
- subscribe
- ping
- unsubscribe
- filterPush
- filterTestUtils
Makes sense?